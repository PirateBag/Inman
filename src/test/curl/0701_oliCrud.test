Create a new order for W-001 in thhe open state, which should present components also.

===JSON===

{ "rows" : [
        {  "activityState" : "INSERT", "itemId" : "1", "parentOliId" : "0", "quantityOrdered" : "10", "quantityAssigned" : "0.0",
        "startDate" : "2025-0801",  "orderState" : "OPEN",
        "orderType" : "MO" }
] }