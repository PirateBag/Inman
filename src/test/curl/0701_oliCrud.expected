{
  "responseType" : "MULTILINE",
  "errors" : [ {
    "key" : 1,
    "message" : "Added 8 line items to order 6"
  } ],
  "data" : [ {
    "id" : 6,
    "crudAction" : "INSERT",
    "itemId" : 1,
    "quantityOrdered" : 10.0,
    "quantityAssigned" : 0.0,
    "startDate" : "2025-0801",
    "completeDate" : "2025-0803",
    "parentOliId" : 0,
    "orderState" : "OPEN",
    "orderType" : "MOHEAD"
  } ]
}